nova/static/js/wave/volumeRaycaster.js

```diff
@@ -978,9 +978,9 @@ Core.prototype.setTransferFunctionByImage = function(image) {
     texture.flipY = true;
     texture.needsUpdate = true;
     if(this._mode == "3d") {
-    this._secondPassSetUniformValue("uTransferFunction", texture);
+        this._secondPassSetUniformValue("uTransferFunction", texture);
     }
-    this.onChangeTransferFunction.call(image);
+    //this.onChangeTransferFunction.call(image);
```

Comment:
No, remove this line.

Reply:
I don't like to comment out code like this either, but these are generated builds and should be addressed in Wave core, not Nova. Even if we remove this line, there are still so many other code lines that have been commented out. What I can do is use a minified version of volumeRaycaster.js

Let me know if its alright and i'll do that

Reply:
No, ignore this one here then.